fixtures/mega-lite/framebot-600.json

```diff
@@ -0,0 +1,854 @@
+        "type": "Focus",
+        "distanceStart": "far",
+        "distanceEnd": "near"
+      }
+    },
+    "Iris": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 127],
+          "type": "Iris",
+          "openPercentStart": "open",
+          "openPercentEnd": "closed"
+        },
+        {
+          "dmxRange": [128, 135],
+          "type": "Iris",
+          "openPercent": "closed"
+        },
+        {
+          "dmxRange": [136, 173],
+          "type": "IrisEffect",
+          "effectName": "Pulse Closed to Open",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [174, 211],
+          "type": "IrisEffect",
+          "effectName": "Pulse Open to Closed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [212, 249],
+          "type": "IrisEffect",
+          "effectName": "Iris Bounce",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [250, 255],
+          "type": "Iris",
+          "openPercent": "closed"
+        }
+      ]
+    },
+    "Frost": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [10, 255],
+          "type": "Frost",
+          "frostIntensityStart": "off",
+          "frostIntensityEnd": "high"
+        }
+      ]
+    },
+    "Control": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 19],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [20, 29],
+          "type": "Maintenance",
+          "comment": "Reset All",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [30, 39],
+          "type": "Maintenance",
+          "comment": "Reset Pan/Tilt",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [40, 49],
+          "type": "Maintenance",
+          "comment": "Reset Color/Gobo/CMYK",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [50, 59],
+          "type": "Maintenance",
+          "comment": "Reset Zoom/Focus/Prism/Frost",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [60, 69],
+          "type": "Maintenance",
+          "comment": "Reset Colormation/Iris/Framing",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [70, 79],
+          "type": "Maintenance",
+          "comment": "Display Default",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [80, 89],
+          "type": "Maintenance",
+          "comment": "Display Off",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [90, 255],
+          "type": "NoFunction"
+        }
+      ]
+    },
+    "Frame 1A": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
```

Comment:
Please use [`BladeInsertion`](https://github.com/OpenLightingProject/open-fixture-library/blob/master/docs/capability-types.md#blade-insertion) for these capabilities.